packages/host/app/components/operator-mode/detail-panel-selector.gts

```diff
@@ -105,9 +123,35 @@ export default class Selector extends Component<Signature> {
                   }}
                   disabled={{selectorItem.disabled}}
                 >
-                  <span class='selector-item'>
-                    {{selectorItem.text}}
-                  </span>
+                  <div class='selector-item'>
+                    {{#if selectorItem.declaration.exportedAs}}
+                      <span class='exported-arrow'>
+                        <DiagonalArrowLeftUp
+                          width='20'
+                          height='20'
+                        />
+                      </span>
+                      <span
+                        class='exported'
+                      >{{selectorItem.declaration.exportedAs}}</span>
+                      {{#unless
+                        (eq
+                          selectorItem.declaration.exportedAs
+                          selectorItem.declaration.localName
+                        )
+                      }}<span
+                        >({{selectorItem.declaration.localName}})</span>{{/unless}}
+                    {{else}}
+                      <span class='non-exported'>{{if
+                          selectorItem.declaration.localName
+                          selectorItem.declaration.localName
+                          '??'
```

Comment:
Was the `??` a placeholder? Is there a case where there's no name?

Reply:
Yea we should replace this with something that makes sense. [No Name Found] as previously